portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,28 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function displayAbout() {
+    const title = document.getElementById('content-title');
+    title.innerText = "Here is some background information about me:";
+    $(function(){
+        $("#content-panel").load("pages/aboutme.html"); 
```

Comment:
cool, you moved this section to a separate file!

Reply:
thanks!